flocker/acceptance/endtoend/test_dockerplugin.py

```diff
@@ -145,6 +145,17 @@ def _create_volume(self, client, name, driver_opts):
         """
         result = client.create_volume(name, u'flocker', driver_opts)
         self.addCleanup(client.remove_volume, name)
+
+        # The docker volume API doesn't block.
+        # Wait until the volume has been mounted.
```

Comment:
I think we should consider renaming this function with this addition. It used to just be a light wrapper on the underlying create_volume API, but now that you added the wait until, perhaps rename this mount_volume or at least change the docstring?